policy_network/HiveNet.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch import nn
+from torch.nn import functional as F
+from hive_vision import HiveNetVision
+
+
+class HiveNet(nn.Module):
+
+    def __init__(self, kernel_size, stride, num_of_clusters):
+        super(HiveNet, self).__init__()
+        self.vision = HiveNetVision(kernel_size, stride, outputs=32)
+        self.policy_hidden1 = nn.Linear(32+num_of_clusters, 64)
```

Comment:
Jeśli te wartości są jakoś związane z liczbą wejść to proponuję je pozamieniać na outputs * 2 itd. Jeśli to są liczby, które są przypadkiem wielokrotnościami i może tam być + 21 czy coś to poproszę o enuma albo zmienne w stylu FIRST_LAYER_NEURONS_PER_CLUSTER. To samo wszędzie indziej.

Reply:
Poprawki juz wprowadzone na nowym commicie